src/transforms/groupby.js

```diff
@@ -35,14 +35,24 @@ exports.attributes = {
         ].join(' ')
     },
     style: {
-        valType: 'any',
-        dflt: {},
-        description: [
-            'Sets each group style.',
-            'For example, with `groups` set to *[\'a\', \'b\', \'a\', \'b\']*',
-            'and `style` set to *{ a: { marker: { color: \'red\' } }}',
-            'marker points in group *\'a\'* will be drawn in red.'
-        ].join(' ')
+        target: {
+            valType: 'any',
```

Comment:
Maybe weird, maybe not:

<img width="172" alt="screen shot 2017-03-10 at 12 18 01" src="https://cloud.githubusercontent.com/assets/572717/23805396/ac0aea9e-058b-11e7-9d37-c6aeb8eeaefb.png">